core/geometry/src/geometry3d/AngleSweep.ts

```diff
@@ -307,25 +388,28 @@ export class AngleSweep implements BeJSONFunctions {
    * @return {*} {degrees: [startAngleInDegrees, endAngleInDegrees}
    */
   public toJSON(): any {
-    // return { degrees: [this.startDegrees, this.endDegrees] };
     return [this.startDegrees, this.endDegrees];
   }
-  /** test if start and end angles match, with explicit name to clarify that there is no test for 360-degree shifts. */
+  /** test if start and end angles match with radians tolerance.
+   * * Period shifts are allowed.
+   */
   public isAlmostEqualAllowPeriodShift(other: AngleSweep): boolean {
     return Angle.isAlmostEqualRadiansAllowPeriodShift(this._radians0, other._radians0)
-      && Angle.isAlmostEqualRadiansNoPeriodShift(this._radians1 - this._radians0, other._radians1 - other._radians0);
+      && Angle.isAlmostEqualRadiansAllowPeriodShift(this._radians1 - this._radians0, other._radians1 - other._radians0);
   }
```

Comment:
We need to confirm with Earlin, but I believe `AngleSweep` "equality" means the start angle is optionally allowed to shift by 2pi, but _never_ the sweep. So I'm thinking the implementation should remain untouched, but please clarify this subtlety in the documentation.

Reply:
will ask him